google-cloud-bigquery/lib/google/cloud/bigquery/service.rb

```diff
@@ -39,15 +39,17 @@ class Service
         attr_accessor :credentials
 
         # @private
-        attr_reader :retries, :timeout
+        attr_reader :retries, :timeout, :endpoint
```

Comment:
We are calling this value `host` on other service classes.

Reply:
Okay, changed. We've been instructed to use the term `endpoint` for the public interfaces (e.g. parameters to `Bigquery.new` and the name of the config key) because documentation wants to use common terminology across languages. But since `Service` is an internal class, we can use the name `host` there.